libqtile/backend/x11/window.py

```diff
@@ -1315,6 +1523,9 @@ def floating(self, do_float):
             else:
                 # if we are setting floating early, e.g. from a hook, we don't have a screen yet
                 self._float_state = FloatStates.FLOATING
+            if not self.kept_above and self.qtile.config.floats_kept_above:
+                print(f"1503 - Setting keep above: {self.name}")
```

Comment:
Oops :)

Reply:
This is why I have a test in `qtile-extras` that fails if it finds print statements in my PRs!!

Reply:
Still there :)

Reply:
> This is why I have a test in qtile-extras that fails if it finds print statements in my PRs!!

Fancy copy-pasting it into this repo? ;)